salt/config/__init__.py

```diff
@@ -2919,7 +2919,7 @@ def is_profile_configured(opts, provider, profile_name, vm_=None):
         required_keys.append('image')
         if driver == 'vmware':
             required_keys.append('datastore')
-    elif driver in ['linode', 'virtualbox', 'vmware']:
```

Comment:
@cro `clonefrom` should be the default. Unless `image` is specified, `clonefrom` should be a required field.